tests/Resources/Common.robot

```diff
@@ -103,4 +103,9 @@ Verify NPM Version
     ...    against an expected version in a given pod/container
     [Arguments]  ${library}  ${expected_version}  ${pod}  ${namespace}  ${container}=""  ${prefix}=""  ${depth}=0
     ${installed_version} =  Run  oc exec -n ${namespace} ${pod} -c ${container} -- npm list --prefix ${prefix} --depth=${depth} | awk -F@ '/${library}/ { print $2}'
-    Should Be Equal  ${installed_version}  ${expected_version}
\ No newline at end of file
+    Should Be Equal  ${installed_version}  ${expected_version}
+
+Get Cluster Name From Console URL
+    [Documentation]    Get the cluster name from the Openshitf console URL
+    ${name}=    Split String    ${OCP_CONSOLE_URL}        .
```

Comment:
as alternative, you could fetch the URL from the cluster itself

Reply:
Yeah i am aware of that just didn't want to complicate things since we read our variable from ymal file, We can use below commands:

1. oc whoami --show-console
2. oc get routes -n openshift-console | grep -i 'https' | awk '{print $2}'

there could be more. i am not ware of that